README.md

```diff
@@ -62,6 +63,21 @@ generated using a supported Thrift compiler.
 
 *Note:* Currently only generated Python files are supported.
 
+### Black
+
+Uses the [Black](http://checkstyle.sourceforge.net/) opinionated code formatter
+to normalize the format of Python code.
+
+```json
+{
+    "type": "black",
+    "include": "(\\.py$)",
+    "black.linelen": 132
+}
+```
+
+Note: It is recommended to place black before other linters
```

Comment:
You can customize the linter's priority by overriding the `getLinterPriority()` method.